application/tests/unit/test_roaddistance.py

```diff
@@ -1,17 +1,26 @@
 import os
 import json
 from application.main import RoadDistance
+from application.rdlogger import RDLogger
 
 
 class TestRoadDistance:
 
     road_distance = RoadDistance({})
     json_path: str = "tests/unit/test_json/"
 
+    def test_valid_request(self):
+        json_content: dict = self.__fetch_json("dos_road_distance_api_happy.json")
+        road_distance = RoadDistance(json_content)
```

Comment:
Are we pulling in an instance of the logger class just to call it's purge method? Why don't we just shift out a separate os-remove call into a helper class?